src/unity/python/turicreate/toolkits/image_classifier/_evaluation.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+"""
+Class definition and utilities for the evaluation of the image classification model
+"""
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+from ...visualization import _get_client_app_path
+
+import subprocess as __subprocess
+import thread as _thread
+import json as _json
+import base64 as _base64
+import sys as _sys
+import math as _math
+
+class Evaluation(dict):
```

Comment:
I think we had talked about the Evaluation object being returned from all models (not just image classifier). Since we are adding a new API method (explore?) we should make all models return an Evaluation, and have explore throw NotImplementedException in the case of non-image-classifiers (for now).

If that's too major of a refactor, it can be done after this PR goes in. Please either address it in this PR, or open a GitHub issue to reconcile the API return values from evaluate for other models so we don't lose track.

Reply:
issue #1352 need to resolve.